src/g_shared/a_quake.cpp

```diff
@@ -52,6 +52,9 @@ DEarthquake::DEarthquake (AActor *center, int intensityX, int intensityY, int in
 	m_WaveSpeedX = (float)waveSpeedX;
 	m_WaveSpeedY = (float)waveSpeedY;
 	m_WaveSpeedZ = (float)waveSpeedZ;
+	m_Falloff = falloff << FRACBITS;

```

Comment:
Is there some reason this is internally a fixed point number but not exposed as such on the input?